browser/ui/webui/brave_wallet/wallet_page_ui.cc

```diff
@@ -62,6 +62,10 @@ WalletPageUI::WalletPageUI(content::WebUI* web_ui)
       std::string("frame-src ") + kUntrustedTrezorURL + " " +
           kUntrustedLedgerURL + ";");
   source->AddString("braveWalletTrezorBridgeUrl", kUntrustedTrezorURL);
+  source->OverrideContentSecurityPolicy(
```

Comment:
i think this needs to be merged with the frame-src CSP above or else it will break the ledger/trezor integration again. cc @nvonpentz